File: java/junit5/src/main/java/Game/Game.java
package Game;

public class Game {

  private boolean tickCalled;

  public void start() {


  }

  public String view() {
    if (tickCalled) {
      return """
        ---
        --x
        ---""";
    }
    return """
        ---
        -x-
        ---""";
  }

  public void tick() {
    tickCalled = true;
  }
}

File: kotlin/junit/src/test/kotlin/com/kata/Game.kt
package com.kata

class Game {
    fun tick() {
        TODO("Not yet implemented")
    }

    val score = 0
    val board: List<List<FieldState>> = (0..20).map { y ->
        (0..10).map { x -> FieldState.Empty }
    }
}

sealed class FieldState {
    object Empty : FieldState()
}
